lib/instrumentation/transaction.js

```diff
@@ -207,12 +207,16 @@ Transaction.prototype.setDefaultNameFromRequest = function () {
   this.setDefaultName(req.method + ' ' + path)
 }
 
-Transaction.prototype.end = function () {
+Transaction.prototype.end = function (result) {
   if (this.ended) {
     this._agent.logger.debug('tried to call transaction.end() on already ended transaction %o', {id: this.id})
     return
   }
 
+  if (typeof result !== 'undefined') {
```

Comment:
I think `result !== undefined` is marginally easier to read and also doesn't need to create a string (though I expect v8 is probably pretty good at optimzing away any difference between the two versions).

Reply:
fixed